{
  "currentUrl": "https://192.168.5.17:20145/index.html#/users",
  "testError": {
    "matcherName": "",
    "message": "Error: Promise was rejected with the following reason: Error: timeout",
    "stack": "Error: Promise was rejected with the following reason: Error: timeout\n    at timer.catch.e (/Users/danny/Documents/script/node/Sarah-auto/misc/misc_auto_2.2_new/node_modules/webdriverio/build/commands/browser/waitUntil.js:71:11)",
    "passed": false,
    "expected": "",
    "actual": ""
  },
  "browserErrorlog": [
    {
      "level": "SEVERE",
      "message": "https://192.168.5.17:20145/favicon.ico - Failed to load resource: the server responded with a status of 404 (Not Found)",
      "source": "network",
      "timestamp": 1556089844453
    }
  ]
}